week-1-ts-ecosystem/w01e03-object-oriented-structure/src/feature/cart/components/cart.ts

```diff
@@ -0,0 +1,76 @@
+import { Product } from './product';
+
+interface Cart {
+  products: Product[];
+  cartType: 'Auction' | 'Buy Now' | 'Free';
```

Comment:
tego zapisu:
```
'Auction' | 'Buy Now' | 'Free'
```
używasz w kilku miejscach.

Co więcej reprezentuje on rodzaj elementów jakie można dodać per koszyki.
Więc to dość istotne.

Zasługuje na oddzielny plik i `export` typu np:.

```typescript
// cart-type.ts
export type CartType = 'Auction' | 'Buy Now' | 'Free';
```